docs/Introduction.md

```diff
@@ -35,7 +35,7 @@ RedStone was designed with a modular architecture making it easy to incorporate
 
 - RedStone secures [billions of dollars](https://defillama.com/oracles/RedStone?borrowed=true&doublecounted=true) of value in Web 3.0
 - Data Integrity is fundamental to RedStone and is ensured from start to finish (from source to smart contract)
-- There are [3 different ways](./get-started/selecting-redstone-model) to integrate our service tailored to your needs
+- There are [3 different ways](../get-started/selecting-redstone-model) to integrate our service tailored to your needs
```

Comment:
Are you sure? To me it looks like the previous version was correct.

Reply:
I've added 'trailingSlash' parameter. While building with it I had build errors regarding those links. I had to update them. They work, otherwise the build wouldn't go through.